### DISCUSSION TITLE
Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by mymain69** [d2phap/ImageGlass#45] [General] [answered]
I tried looking for it, maybe I missed it but if its not in there can you add keyboard shortcuts on the next update. Ctrl W closes the window and customizable shortcut keys.

Once again thank you for this program.
Can you give us an estimation when the next update that fixes the previous bugs will be available?

**Comment by d2phap** [comment, marked as answer]
Hi @mymain69 
All shortcuts you can find in Main menu (the last button at top right coner). 
ImageGlass is driven to have customization posibility. I will update it for next few releases.

From v3.0, I will try to publish new update more often. Perhaps every month or two (It still depends on my free time ^^).
Thank you
